### ISSUE TITLE
Portrait (vs.) Landscape Checkin Scree

### ISSUE THREAD

**Issue by LifePastor** [petereit/CheckinAir#23]
Hello,

Is it possible for Check-In Air to rotate to the Portrait position on an iPad for checkin verses the Horizontal position?  The Kiosks that we bought to hold our ipads for checkin are only usable in the Portrait position.  

The CheckinAir app for ios automatically rotates to the position of the iPhone but the iPad add only seems to be fixed in the Landscape mode.  Any suggestions?  Thanks so much.

**Comment by petereit** [comment]
Yes. That’s actually a pretty simple correction. I’ll see if I can update the app this weekend.

**Comment by LifePastor** [comment]
Thanks so much Mark.  Please let me know when it’s updated. 

J. Huffman
Life@LifePastor.org
512-660-0166 cell

**Comment by LifePastor** [comment]
Hello Mark,

I was checking to see if you’ve been able to update the Checkin Air app on the iPad to autorotate to portrait mode?   Our event is next week and as of now I’m unable to use iPad checkin because our Kiosks are portrait mode only.  

Thanks so much!

J. Huffman
Life@LifePastor.org
512-660-0166 cell

**Comment by petereit** [comment]
Yes. I was waiting on some information from Breeze on another issue, but I’ll go ahead and push the portrait fix tonight. 

Mark

**Comment by LifePastor** [comment]
Thank you SO much.  I really appreciate your gift to the Kingdom by building this app.  

J. Huffman
Life@LifePastor.org
512-660-0166 cell

**Comment by petereit** [comment]
Version 1.35 that supports iPad portrait orientation has been pushed to the Apple App Store. I'll let you know when Apple has approved it and it is available for update.

**Comment by LifePastor** [comment]
Thanks Mark. 

J. Huffman
Life@LifePastor.org
512-660-0166 cell

**Comment by petereit** [comment]
v 1.35 has been approved and is now available for update.

**Comment by LifePastor** [comment]
Thanks Mark!

J. Huffman
Life@LifePastor.org
512-660-0166 cell
